building/devices/index.rst

```diff
@@ -12,6 +12,6 @@ Device specific information
     hikey960
     juno
     qemu
+    zynqmp
```

Comment:
Please move this down to the end so we keep the platforms sorted in alphabetic order.